airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -493,3 +496,39 @@ def get_container(self, container_list):
                     'container "name" must be provided in "containerOverrides" configuration'
                 )
         raise KeyError(f"No such container found by container name: {self.container_name}")
+
+    def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[TaskInstance]:
```

Comment:
Stupid question again, but how is this being called?

Reply:
It gets called by the scheduler [here](https://github.com/apache/airflow/blob/main/airflow/jobs/scheduler_job_runner.py#L1640) when it makes an adoption pass.